packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -871,6 +872,11 @@ class MenuItemButton extends StatefulWidget {
   /// An optional icon to display after the [child] label.
   final Widget? trailingIcon;
 
+  /// Determines if the menuAnchor should be closed after a [MenuItemButton]
+  /// has been pressed.
```

Comment:
Add a blank doc comment line after this to separate the paragraphs for Dartdoc (it uses the first paragraph as a short summary for the property).

Reply:
done